pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -118,19 +134,42 @@ func (o *MustGatherOptions) Complete(f kcmdutil.Factory, cmd *cobra.Command, arg
 	return nil
 }
 
-func (o *MustGatherOptions) resolveMustGatherImage() (string, error) {
+func (o *MustGatherOptions) resolveImageStreamTagString(s string) (string, error) {
+	namespace, name, tag := parseImageStreamTagString(s)
+	if len(namespace) == 0 {
+		namespace = "openshift"
+	}
+	if len(tag) == 0 {
+		tag = "latest"
+	}
+	return o.resolveImageStreamTag(namespace, name, tag)
+}
+
+func parseImageStreamTagString(s string) (string, string, string) {
+	var imageStreamRegExp = regexp.MustCompile(`((?P<namespace>[^/]*)/)?(?P<name>[^:]*)(:(?P<tag>.*))?`)
```

Comment:
library-go should have a parsing helper.

Reply:
Done